tests/test_sampling.py

```diff
@@ -0,0 +1,23 @@
+from pyriemann.datasets.sampling import (sample_gaussian_spd,
+                                         generate_random_spd_matrix)
+from conftest import is_positive_definite
+import numpy as np
+
+
+def test_sample_gaussian_spd():
+    """Test Riemannian Gaussian sampling."""
+    n_dim = 16
+    n_matrices = 50
+    mean = np.eye(n_dim)
+    sigma = 2.00
+    X = sample_gaussian_spd(n_matrices, mean, sigma, random_state=None)
+    assert X.shape == (n_matrices, n_dim, n_dim)  # X shape mismatch
+    assert is_positive_definite(X)  # X is an array of SPD matrices
+
```

Comment:
I like this a lot, thanks !

Reply:
Checks are red, because this new test fails: `assert 5.0146714087193685 < nan`

@plcrodrigues , is it possible that a too high dispersion `sigma` could generate badly conditioned matrices?

To catch that, a warning could be raised if matrices are not SPD: a test could be added (just before return) on the output of public simulation functions (`generate_random_spd_matrix` and `sample_gaussian_spd`).

Reply:
Yeah, in fact when we have too large `sigma` the `r_parameters` used for generating the SPD matrices start getting very big and very small (like -30 and +30). Since we then take their `exp` for the eigenvalues, we end up with matrices which are very badly conditioned. Ideally, we would want a regularizer à la `lwf` but I don't have any references for doing this. Do you guys think that simply raising a warning and letting the user set a smaller `sigma` would be enough?

Reply:
I think that it is difficult to find a threshold for `sigma`, since matrix conditioning must depends on `n_dim` too.
Adding a warning _a posteriori_ seems a good solution.